injector/disk_failure.go

```diff
@@ -0,0 +1,104 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2023 Datadog, Inc.
+
+package injector
+
+import (
+	"os/exec"
+	"strconv"
+	"strings"
+
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	"github.com/DataDog/chaos-controller/types"
+	"go.uber.org/zap"
+)
+
+type DiskFailureInjector struct {
+	spec   v1beta1.DiskFailureSpec
+	config DiskFailureInjectorConfig
+}
+
+// DiskFailureInjectorConfig is the disk pressure injector config
+type DiskFailureInjectorConfig struct {
+	Config
+	Cmd BPFDiskFailureCommand
+}
+
+type BPFDiskFailureCommand interface {
+	Run(pid int, path string) error
+}
+
+type bPFDiskFailureCommand struct {
+	log *zap.SugaredLogger
+}
+
+const EBPFDiskFailureCmd = "bpf-disk-failure"
+
+func (d bPFDiskFailureCommand) Run(pid int, path string) (err error) {
```

Comment:
I think it's good enough for now as a solution but in the future I would create a generic eBPF program runner we can re-use in other eBPF disruptions to come and that we can mock easily.

Reply:
Good point! I will be really excited to work on this 👍